tm2/pkg/sdk/auth/keeper.go

```diff
@@ -156,6 +156,18 @@ func (ak AccountKeeper) GetNextAccountNumber(ctx sdk.Context) uint64 {
 	return accNumber
 }
 
+func (ak AccountKeeper) FeeCollectorAddress(ctx sdk.Context) crypto.Address {
+	feeCollector := ak.GetParams(ctx).FeeCollector
+	if feeCollector.IsZero() {
+		feeCollector = crypto.AddressFromPreimage([]byte(DefaultFeeCollectorName))
```

Comment:
Just panic if there is no fee collector. Your current implementation isn't caching the case of IsZero. Let's explicitly panic so that blockchain builders can define something custom or just keep the default value, which should work.

Reply:
I'm fine with panicing, but I'm concerned about the fact that `NewAccountKeeper` doesn't initialize `DefaultParam`, which means that by default, calling `FeeCollectorAddress` on a fresh account keeper from `NewAccountKeeper` will panic.